src/core/AutomatableModel.cpp

```diff
@@ -275,9 +275,10 @@ float AutomatableModel::inverseScaledValue( float value ) const
 
 QString AutomatableModel::displayValue( const float val ) const
 {
+	const FloatModel *floatmodel = dynamic_cast<const FloatModel*>( this );
 	switch( m_dataType )
 	{
-		case Float: return QString::number( castValue<float>( scaledValue( val ) ) );
+		case Float: return QString::number( castValue<float>( scaledValue( floatmodel->getRoundedValue() ) ) );
```

Comment:
Yes, this is stupid. My intention was to fix the mixer bug ( 1.2345 % means 123,45 % ). I guess I revert this and search an other solution for this.